src/browser-lib/parcel.component.ts

```diff
@@ -0,0 +1,142 @@
+import { Component, OnInit, OnDestroy, Input, OnChanges, ViewChild, ElementRef } from '@angular/core';
+
+
+@Component({
+  selector: 'parcel',
+  template: `
+  <div #parcelDiv>Parcel works!</div>
```

Comment:
This is the wrong template we should not be putting "Parcel works!" into the dom

Reply:
I know it, it's just as a debugger